sdks/python/test-suites/dataflow/py2/build.gradle

```diff
@@ -39,6 +39,8 @@ task preCommitIT(dependsOn: ['sdist', 'installGcpTest']) {
     def precommitTests = [
         "apache_beam.examples.wordcount_it_test:WordCountIT.test_wordcount_it",
         "apache_beam.examples.streaming_wordcount_it_test:StreamingWordCountIT.test_streaming_wordcount_it",
+        "apache_beam.examples.streaming_wordgroup_it_test:StreamingWordGroupIT.test_discard_late_data",
```

Comment:
Are these tests passing? According to the JIRA they do not?

Reply:
It was my understanding that Dataflow integration tests are run as part of the "Run Python PreCommit" test suite (although I don't fully understand the gradle setup yet)? That test suite contains the following logs, which I thought indicated that the tests pass on Dataflow?

```
> Task :sdks:python:test-suites:dataflow:py2:preCommitIT
test_discard_late_data (apache_beam.examples.streaming_wordgroup_it_test.StreamingWordGroupIT) ... ok
test_single_output_per_window (apache_beam.examples.streaming_wordgroup_it_test.StreamingWordGroupIT) ... ok
```

```
> Task :sdks:python:test-suites:dataflow:py37:preCommitIT
Found: https://console.cloud.google.com/dataflow/jobsDetail/locations/us-central1/jobs/2019-08-01_21_57_33-3043197201172537996?project=apache-beam-testing.
Found: https://console.cloud.google.com/dataflow/jobsDetail/locations/us-central1/jobs/2019-08-01_21_57_35-16912425374424101477?project=apache-beam-testing.
Found: https://console.cloud.google.com/dataflow/jobsDetail/locations/us-central1/jobs/2019-08-01_22_06_51-9426310355159334842?project=apache-beam-testing.
Found: https://console.cloud.google.com/dataflow/jobsDetail/locations/us-central1/jobs/2019-08-01_22_15_35-10554830283046206157?project=apache-beam-testing.
test_streaming_wordcount_it (apache_beam.examples.streaming_wordcount_it_test.StreamingWordCountIT) ... ok
test_wordcount_it (apache_beam.examples.wordcount_it_test.WordCountIT) ... ok
test_discard_late_data (apache_beam.examples.streaming_wordgroup_it_test.StreamingWordGroupIT) ... ok
test_single_output_per_window (apache_beam.examples.streaming_wordgroup_it_test.StreamingWordGroupIT) ... ok
```

The tests definitely pass on Dataflow when I try them out locally.